### ISSUE TITLE
Error during dotnet tool install

### ISSUE THREAD

**Issue by ghost** [gabrielweyer/xunit-to-junit#6]
Hello, thank you a lot for the nice tooling.
I'm trying to install dotnet-xunit-to-junit dotnet tool, but receive an error: 
```
dotnet tool install dotnet-xunit-to-junit
warning NU3028: Package 'dotnet-xunit-to-junit 2.0.2' from source 'https://api.nuget.org/v3/index.json': The repository primary signature's timestamp found a chain building issue: ExplicitDistrust: The trust setting for this policy was set to Deny.
error NU3037: Package 'dotnet-xunit-to-junit 2.0.2' from source 'https://api.nuget.org/v3/index.json': The repository primary signature validity period has expired.
The tool package could not be restored.
Tool 'dotnet-xunit-to-junit' failed to install. 
```
I suppose there is something wrong with the dotnet-xunit-to-junit as other tools install successfully .

As a workaround I played with trusted certificates settings in [Nuget.config](https://gist.github.com/andrey-covergo/688bd8c47b33013584bc694d508c49df) and was able to install dotnet-xunit-to-junit tool, but wonder why it failed initially. Certificate fingerprints are taken from `dotnet tool install dotnet-xunit-to-junit -v diag` output

**Comment by gabrielweyer** [comment]
Hey, are you running on a Mac? Which .NET runtime are you using?

I can't reproduce this on Windows 10. I opened the package in the NuGet Package Explorer and the signature is marked as valid:

![image](https://user-images.githubusercontent.com/2101647/134329072-c536cc67-bcea-4546-b3ef-9836d77732d1.png)

**Comment by andreyleskov** [comment]
Yes, I run it on Mac, .net5. Maybe it is my local issue. Need to think about how to diagnose.

**Comment by ghost** [comment]
I confirm it is my local Mac-specific issue, as the tool installs properly in a Docker environment at the same machine. Sorry for the disruption.

**Comment by gabrielweyer** [comment]
I recently published [3.0.0](https://www.nuget.org/packages/dotnet-xunit-to-junit/3.0.0). Maybe because the package has been published recently it would fix the issue on your Mac?

**Comment by ghost** [comment]
Hey @gabrielweyer, thanks a lot for the update. I was able to install the tool by standard way `dotnet tool install dotnet-xunit-to-junit`
